uit/gui_tools/submit.py

```diff
@@ -184,19 +184,33 @@ def update_file_path(self, _, index):
         else:
             self.env_values[index].value = self.file_browser.value[0]
 
+    def get_versions(self, uit_client):
```

Comment:
I'm not sure that it makes sense to have this function here. The PyUIT library doesn't know/care about versions. It should be implemented in a subclass that is software specific.